src/rtp/id.rs

```diff
@@ -11,13 +11,17 @@ use rand::random;
 use crate::io::Id;
 
 macro_rules! str_id {
-    ($id:ident, $name:literal, $num:tt) => {
+    ($id:ident, $name:literal, $num:tt, $new_len:tt) => {
         #[derive(Clone, Copy, PartialEq, Eq, Hash, Serialize, Deserialize)]
         pub struct $id([u8; $num]);
 
         impl $id {
             pub fn new() -> $id {
-                $id(Id::random().into_array())
+                let mut arr = Id::<$num>::random().into_array();
```

Comment:
This seems weird? Why are we removing a bunch of the random data here?

Reply:
We want 3-letter Mid (like FF). Xnorpx/Peter wanted the possibility to have longer Mid. When we changed to the longer ones, we also made our random Mids very long. I want to bring the default random ones back to 3.